README.md

```diff
@@ -1,5 +1,6 @@
 ![](./kani-logo.png)
-![Regressions with latest CBMC](https://github.com/zhassan-aws/kani/actions/workflows/cbmc-latest.yml/badge.svg)
+![Kani regression](https://github.com/zhassan-aws/kani/actions/workflows/kani.yml/badge.svg)
```

Comment:
I hadn't noticed that I added my fork instead of origin! Can you update them?

Reply:
ooops... yes, let me fix that!